app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -41,17 +42,35 @@ class Filter extends \Magento\Backend\Block\Widget\Grid\Extended
     ];
 
     /**
+     * @var \Magento\ImportExport\Model\Export\ConfigInterface
+     */
+    private $exportConfig;
+
+    /**
+     * @var \Magento\CatalogImportExport\Model\Export\Product\Type\Factory
+     */
+    private  $typeFactory;
+
+    /**
+     * Filter constructor.
+     * @param \Magento\ImportExport\Model\Export\ConfigInterface $exportConfig
+     * @param \Magento\CatalogImportExport\Model\Export\Product\Type\Factory $typeFactory
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Backend\Helper\Data $backendHelper
      * @param \Magento\ImportExport\Helper\Data $importExportData
      * @param array $data
      */
+
     public function __construct(
+        \Magento\ImportExport\Model\Export\ConfigInterface $exportConfig = null,
```

Comment:
Can you please move the new added dependencies to the end of constructor parameters

Reply:
sure , adding it